robocorp-code/src/robocorp_code/playwright/robocorp_browser/__init__.py

```diff
@@ -208,20 +254,30 @@ def screenshot(
     Returns:
         The bytes from the screenshot.
     """
-    # import base64
+    import base64
+
+    from robocorp import log
```

Comment:
We should not import `robocorp.log` here (it's one of the reasons for this fork).